### ISSUE TITLE
How to filter with multiple values?

### ISSUE THREAD

**Issue by OvalMedia** [metafizzy/isotope#1418]
**Test case:** https://codepen.io/ovalmedia/pen/JZZJvK

I only put in the HTML in the codepen as I don't know how to properly use the functions to achieve this.
The filters each refer to a category id. The items can have one or multiple category ids (e.g. "2,3,4").

I get when I use functions to filter they either return true or false. But I don't see how inside a function I know what filter button has been clicked. So I don't know what id I am actually looking for.

So basically I would like to apply a filter like this:

`var inCat = function(id) {
	var ids = $(this).attr('data-cat').split(',');
	return (ids.indexOf(id) >= 0 ? true : false)
};`

But I don't know how.

**Comment by desandro** [comment]
Thanks for reporting this issue.

The easiest solution is to break up the data in the `data-cat` attribute into separate classes, `cat1, cat2, cat3`. This way you can use traditional class based filtering and not have to worry about filter function code.
